configure.ac

```
@@ -1443,6 +1442,7 @@ if test "$enable_romio" = "yes" ; then
        if test ! -d lib ; then mkdir lib ; fi
        # tell mpi.h to include mpio.h
        PAC_HAVE_ROMIO
+       PAC_CONFIG_SUBDIR_ARGS([src/mpi/romio],[$subsys_config_args],[],[AC_MSG_ERROR([src/mpi/romio configure failed])])
```

```suggestion
       PAC_CONFIG_SUBDIR_ARGS([src/mpi/romio],[],[],[AC_MSG_ERROR([src/mpi/romio configure failed])])
```
ROMIO doesn't take extra configure args. If needed, we'll specifically add here for clarity.